drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java

```diff
@@ -761,18 +762,13 @@ public boolean hasColumnDefinitions() {
 
     @Override
     public boolean canConditionBeDeleted(final ConditionCol52 col) {
```

Comment:
I noticed deletion of ```BRLConditionColumn``` was incorrectly being handled by ```canConditionBeDeleted( final ConditionCol52 col )``` (as it is a sub-class of it).. so fixed to use the correct check for the applicable class/sub-class.